programs/launcher/main.cpp

```diff
@@ -478,43 +623,47 @@ launcher_def::write_config_file (eosd_def &node) {
     }
     cfg << "plugin = eosio::producer_plugin\n";
   }
+  if( instance.has_db ) {
+    if( !node.producers.size() ) {
+      cfg << "plugin = eosio::producer_plugin\n";
+    }
+    cfg << "plugin = eosio::db_plugin\n";
+  }
   cfg << "plugin = eosio::chain_api_plugin\n"
       << "plugin = eosio::wallet_api_plugin\n"
-      << "plugin = eosio::db_plugin\n"
       << "plugin = eosio::account_history_plugin\n"
       << "plugin = eosio::account_history_api_plugin\n";
   cfg.close();
-  if (!node.on_host()) {
-    prep_remote_config_dir (node);
+  if (!host->is_local()) {
+    prep_remote_config_dir (instance);
     string scp_cmd_line = network.ssh_helper.scp_cmd + " ";
-    const string &args = node.ssh_args.length() ? node.ssh_args : network.ssh_helper.ssh_args;
+    const string &args = host->ssh_args.length() ? host->ssh_args : network.ssh_helper.ssh_args;
     if (args.length()) {
       scp_cmd_line += args + " ";
     }
     scp_cmd_line += filename.string() + " ";
 
-    const string &uid = node.ssh_identity.length() ? node.ssh_identity : network.ssh_helper.ssh_identity;
+    const string &uid = host->ssh_identity.length() ? host->ssh_identity : network.ssh_helper.ssh_identity;
     if (uid.length()) {
       scp_cmd_line += uid + "@";
     }
 
-    bf::path dpath = bf::path (node.eos_root_dir) / node.data_dir / "config.ini";
-    scp_cmd_line += node.hostname + ":" + dpath.string();
+    bf::path dpath = bf::path (host->eos_root_dir) / instance.data_dir / "config.ini";
+    scp_cmd_line += host->host_name + ":" + dpath.string();
 
     cerr << "cmdline = " << scp_cmd_line << endl;
-    int res = boost::process::system (scp_cmd_line);
+    int res = 1; //boost::process::system (scp_cmd_line);
```

Comment:
scp command line not actually executed anymore?

Reply:
Yikes! Good catch, update for this on its way.